R/session/vsc.R

```diff
@@ -321,6 +321,7 @@ show_view <- !identical(getOption("vsc.view", "Two"), FALSE)
 if (show_view) {
   get_column_def <- function(name, field, value) {
     filter <- TRUE
+    tooltip <- typeof(value)
```

Comment:
What about 

```r
tooltip <- sprintf(
  "class: [%s], type: %s",
  toString(class(value)),
  typeof(value)
)
```

so that the tooltip contains the value of both `class()` and `typeof()`.